Bottom: 2bc03f59c7d06431e5d829a7405f892d82c93b72
Top:    db29c5457ff0fea504478c5c191ecb56245a39ce
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 19:54:37 +0100

Refresh of hid-gtk-separate-out-board-fli

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 8632e3a..dc97a17 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1186,7 +1186,7 @@ ghid_render_pixmap (int cx, int cy, double zoom, int width, int height, int dept
   glTranslatef (priv->flip_x ?  gport->view_x0 - PCB->MaxWidth  :
                                -gport->view_x0,
                 priv->flip_y ?  gport->view_y0 - PCB->MaxHeight :
-                              -gport->view_y0, 0);
+                               -gport->view_y0, 0);
   region.X1 = MIN(Px(0), Px(gport->width + 1));
   region.Y1 = MIN(Py(0), Py(gport->height + 1));
   region.X2 = MAX(Px(0), Px(gport->width + 1));
